frontends/web/src/routes/settings/settings.tsx

```diff
@@ -272,6 +287,28 @@ class Settings extends Component<Props, State> {
                               {t('settings.expert.electrum.title')}
                             </SettingsButton>
                           </div>
+                          { version !== undefined ? (
+                            <div>
+                              <h3 className="subTitle">{t('settings.info.title')}</h3>
+                              <div className="box slim divide m-bottom-large">
+                                <SettingsItem
+                                  optionalText={version}
+                                  primaryText={t('settings.info.version')}>
+                                  { update !== null && update !== undefined ? (
+                                    <div>
+                                      {downloadElement}
+                                      <RedDot />
+                                    </div>
+                                  ) : (
+                                    <div>
+                                      {t('settings.info.up-to-date')}
+                                      <Checked />
+                                    </div>
+                                  )}
+                                </SettingsItem>
```

Comment:
Maybe it should have an `onClick` on  the SettingsItem component instead of the link, so that the whole area is clickable.

Reply:
Done :+1: